tests/page-objects/default/enketo/pregnancy.wdio.page.js

```diff
@@ -0,0 +1,218 @@
+/**
+ * Terminology.
+ *  lmp: Last Menstrual Period
+ *  lmpApprox: Approximate date of last cycle
+ *  edd: Estimate delivery date
+ *  LLIN: long-lasting insecticidal net
+ */
+
+const moment = require('moment');
+const genericForm = require('../../../page-objects/default/enketo/generic-form.wdio.page');
+const contactPage = require('../../../page-objects/default/contacts/contacts.wdio.page');
+
+const GESTATION_AGE = {lmp: 'method_lmp', lmpApprox: 'method_approx', edd: 'method_edd', none: 'none'};
+const FIRST_PREGNANCY_VALUE = {yes: 'primary', no: 'secondary'};
+
+const FORM = 'form[data-form-id="pregnancy"]';
+const SUMMARY_SECTION = `${FORM} section[name="/pregnancy/summary"]`;
+const KNOW_FUTURE_VISITS = 
```

Comment:
```suggestion
const KNOWN_FUTURE_VISITS = 
```

Reply:
When I created this constant I was not sure if it was `known` or `know` 😅 
Thank you!